src/components/Books.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import Book from './Book';
+
+const Books = () => {
```

Comment:
### [Optional]
Let's follow a good naming conversion.
Let's  change the component to `BookList` and don't forget to update the route